src/Result/LightningNode.php

```diff
@@ -11,13 +11,46 @@ class LightningNode extends AbstractResult
      */
     public function getNodeURIs(): array
     {
-        $data = $this->getData();
-        return $data['nodeURIs'];
+        return $this->getData()['nodeURIs'];
     }
 
     public function getBlockHeight(): int
     {
-        $data = $this->getData();
-        return $data['blockHeight'];
+        return $this->getData()['blockHeight'];
+    }
+
+    public function getAlias(): string
```

Comment:
Hm, maybe I'm on the wrong page here or docs outdated? Don't see those fields here: https://docs.btcpayserver.org/API/Greenfield/v1/#operation/InternalLightningNodeApi_GetInfo

Reply:
Yep the documentation is not updated, those fields exist in my test case. I tried to find where to make a PR to the docs but I can't find it. Happy to update it with the new fields that exist.